CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+[2.8.1] - 2021-09-10
+---------------------
+##### Fixed
+- #286 Form's default media references are removed after submission
+- #286 Image labels not rendered for new forms
```

Comment:
This is #287 FWIW. It probably shouldn't have been closed as a duplicate as it had a different underlying cause. But in both cases it was resolved by #293, which I think should be noted as well.